src/hooks/use-all-media.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import fetchTwitchVideos from '../utils/fetch-twitch-videos';
+import fetchYoutubeData from '../utils/fetch-youtube-data';
+import fetchLybsinPodcasts from '../utils/fetch-lybsin-podcasts';
+import { TWITCH_HEADERS, TWITCH_VIDEO_URL } from '../constants';
+
+const useAllMedia = () => {
+    const [media, setMedia] = useState(null);
+    const [error, setError] = useState(null);
+    const [isLoading, setIsLoading] = useState(false);
+    const YT_PLAYLIST = 'PLN3n1USn4xlmyw3ebYuZmGp60mcENitdM';
+
+    useEffect(() => {
+        const getMedia = async () => {
+            setIsLoading(true);
+            try {
+                const youtubeVideos = fetchYoutubeData(YT_PLAYLIST);
+                const twitchVideos = fetchTwitchVideos(
+                    TWITCH_HEADERS,
+                    TWITCH_VIDEO_URL
```

Comment:
I noticed we actually are using the same url and headers in each callsite, do you think it would make sense to just import these values in `fetchTwitchVideos` so each callsite doesn't have to import them?

Reply:
I agree. That would make things a lot easier.